### PR TITLE
fix: "Mention highlight and cursor hover highlight has different corner radius"

### PR DESCRIPTION
**Pull request by yaya-usman**
Fixes:  [vector-im/element-web#21845](https://github.com/vector-im/element-web/issues/21845)

Issue reported by @HarHarLinks 

Signed-off-by: Yaya Usman [yaya-usman@users.noreply.github.com](mailto:yaya-usman@users.noreply.github.com) 

**Before:** 

https://user-images.githubusercontent.com/38439166/164485431-ad2945b4-1c3b-463a-8607-d9e69e53f6ef.mp4

**After:**

https://user-images.githubusercontent.com/38439166/164485389-e8b18eb0-23c7-4003-8ce3-70f548c549f0.mp4

Type: defect

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * fix: "Mention highlight and cursor hover highlight has different corner radius" ([\#8384](https://github.com/matrix-org/matrix-react-sdk/pull/8384)). Contributed by @yaya-usman.

Preview: https://pr8384--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by HarHarLinks** [comment]
unless it is clear from somewhere unbeknownst to me, design (or their figmas) might have to answer whether the bigger ("ping") or the smaller ("hover") radius is the correct one.

**Comment by yaya-usman** [comment]
> unless it is clear from somewhere unbeknownst to me, design (or their figmas) might have to answer whether the bigger ("ping") or the smaller ("hover") radius is the correct one.

I guess the bigger one should be, it looked fine after the fix. You can reproduce to see

**Comment by andybalaam** [comment]
The right-hand border radius is the larger size.

@yaya-usman a screenshot of the before and after would help here.

**Comment by yaya-usman** [comment]
> The right-hand border radius is the larger size, so I assert that the larger size is correct.
> 
> @yaya-usman a screenshot of the before and after would help here.

I recorded the video of the before and after fix but it was pixelated due to the dimension of the area i wanted to show being small. I am trying to see if i can record a less pixelated one

**Comment by yaya-usman** [comment]
@andybalaam i have included a video in the description but it's not that clean as stated earlier, to spot the difference you can view in fullscreen and notice the corners.

**Review by robintown — COMMENTED** [review]
Thanks, for the record the border radius mismatch existed because of https://github.com/vector-im/element-web/issues/18108, but the way that encryption highlights are rendered has changed since then in https://github.com/matrix-org/matrix-react-sdk/pull/7209, so the custom radius is no longer necessary at all

File: res/css/views/rooms/_EventTile.scss [hunk 1]
```diff
@@ -223,8 +223,7 @@ $left-gutter: 64px;
         overflow-y: hidden;
     }
 
-    &.mx_EventTile_selected .mx_EventTile_line,
-    &:hover .mx_EventTile_line {
+    &.mx_EventTile_selected .mx_EventTile_line {
         border-top-left-radius: 4px;
         border-bottom-left-radius: 4px;
     }
```
[the remark this answers is not in the document]
Comment (yaya-usman) [reply res/css/views/rooms/_EventTile.scss:229]: Yes, removing it works as well. But the chat tile border radius will look a little different and take it's original form. I hope it won't be a problem before i push my changes.

**Without removing the entire block**
Capture2

**after removing the entire block**
Capture3

[the remark this answers is not in the document]
Comment (robintown) [reply res/css/views/rooms/_EventTile.scss:229]: Yes, that's fine, the smaller radius was a workaround for an issue that doesn't exist anymore

**Review by robintown — APPROVED** [review]

**Comment by yaya-usman** [comment]
Thanks @robintown  : )

**Pull request merged**